executor/executor_test.go

```diff
@@ -181,13 +181,23 @@ func (s *testSuite) TestAdmin(c *C) {
 	c.Assert(row.Len(), Equals, 10)
 	txn, err = s.store.Begin()
 	c.Assert(err, IsNil)
-	historyJobs, err := admin.GetHistoryDDLJobs(txn)
+	historyJobs, err := admin.GetHistoryDDLJobs(txn, 10)
 	c.Assert(len(historyJobs), Greater, 1)
 	c.Assert(len(row.GetString(1)), Greater, 0)
 	c.Assert(err, IsNil)
 	c.Assert(row.GetInt64(0), Equals, historyJobs[0].ID)
 	c.Assert(err, IsNil)
 
+	r, err = tk.Exec("admin show ddl jobs 20")
+	c.Assert(err, IsNil)
+	chk = r.NewChunk()
+	err = r.Next(ctx, chk)
+	c.Assert(err, IsNil)
+	row = chk.GetRow(0)
+	c.Assert(row.Len(), Equals, 10)
```

Comment:
Why is its value 10 is not 20?

Reply:
I think it the first-row length.

Reply:
ye~

Reply:
Do we need to check the number of jobs in the result?

Reply:
The number of jobs in the result cannot be ensured, it is related to the actual DDL jobs.